src/doc/unstable-book/src/library-features/range-ensure-subset-of.md

```diff
@@ -0,0 +1,10 @@
+# `range_ensure_subset_of`
+
+The tracking issue for this feature is: [#76393]
+
+------------------------
+
+This adds [`Range::ensure_subset_of`].
+
+[#76393]: https://github.com/rust-lang/rust/issues/76393
+[`Range::ensure_subset_of`]: https://doc.rust-lang.org/std/ops/struct.Range.html#method.ensure_subset_of
```

Comment:
I believe this should link to nightly since this is a nightly-only feature:

```suggestion
[`Range::ensure_subset_of`]: https://doc.rust-lang.org/nightly/std/ops/struct.Range.html#method.ensure_subset_of
```

Reply:
Could also just delete the file, since library features [don't usually add files here](https://github.com/rust-lang/rust/pull/81107) -- they just use auto-generated ones.

Reply:
Can't this be a relative link?

Reply:
@scottmcm I only added it based on [the link in tracking issue templates](https://rustc-dev-guide.rust-lang.org/stabilization_guide.html#documentation-prs). However, that page is a little confusing, so I might be misunderstanding its recommendation.

Since this page is mostly a link to the tracking issue, removing it makes sense.

Reply:
@RalfJung Yes, that's a good idea. However, I removed the page instead since it's not very useful and seems to not be required.